test/db/migrate/old_schema.test.js

```diff
@@ -54,9 +56,27 @@ var gulp    = require("gulp");
       });
     });
 
+    describe(Support.getTestDialectTeaser(flag + " auto migrate old schema"), function () {
+      beforeEach(function (done) {
+        return prepareLegacyScenario.call(this, {
+          autoMigrateOldSchema: true
+        }, function () { done(); });
+      });
+
+      it("shouldn't enforce if we provide autoMigrateOldSchema option", function (done) {
```

Comment:
Could you please add a check before and after the migration, that ensures the previously old table structure is now the new one?